GenderPayGap.WebUI/Views/Home/AccessibilityStatement.cshtml

```diff
@@ -0,0 +1,122 @@
+﻿@using GenderPayGap.Core
+@{
+    ViewBag.Title = "Accessibility Statement - Gender pay gap reporting service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+<div class="govuk-grid-row" xmlns="http://www.w3.org/1999/html">

```

Comment:
I don't think we need the `xmlns` attribute (although let me now if you think we do)